TestON/drivers/common/cli/onosdriver.py

```diff
@@ -173,7 +173,81 @@ def cleanInstall( self ):
                         'Runtime\sEnvironment\sto\scontinue',
                     'BUILD\sFAILURE',
                     'BUILD\sSUCCESS',
-                    'ONOS\$',
```

Comment:
Not at all now.
I had modified a little bit and changed it back to original.

Thank you,

Sangho


On Wed, Apr 1, 2015 at 2:44 PM, Jon Hall <notifications@github.com> wrote:

> In TestON/drivers/common/cli/onosdriver.py
> <https://github.com/opennetworkinglab/ONLabTest/pull/107#discussion_r27615128>
> :
>
> > @@ -173,7 +173,81 @@ def cleanInstall( self ):
> >                          'Runtime\sEnvironment\sto\scontinue',
> >                      'BUILD\sFAILURE',
> >                      'BUILD\sSUCCESS',
> > -                    'ONOS\$',
>
> So I think current master has some changes here that are conflicting with
> yours? You didn't have much changes in cleanInstall did you?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/107/files#r27615128>.
>